microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -30,8 +30,8 @@ This release adds improvements related to the following components and concepts.
 
 //L3 major categories with features in each as L4s, for example:
 [id="microshift-4-15-rhel"]
-=== {op-system-base-full} {op-system-version}
-* {microshift-short} runs on {op-system-base-full} versions 9.2 and 9.3.
+=== {op-system-base-full}
+* {microshift-short} now runs on {op-system-base-full} versions 9.2 and 9.3. Compatibility with {op-system} is an enhancement for {microshift-short} version 4.15
```

Comment:
It dont understand the "compatibility...." sentence. V4.14 also has been compatible with RHEL.

Reply:
Supposed to say "...with 9.3"; added.